frontend/src/util/index.tsx

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+import React from 'react';
+
+const format_date = (
```

Comment:
i think export const would work better here, since we might have other functions in this file